### ISSUE TITLE
compiling dkms in linux-3.19

### ISSUE THREAD

**Issue by Alukardd** [aabc/ipt-netflow#41]
Hello.

I have trouble with compiling ipt_NETFLOW module for linux-3.19.
Because in linux-3.19 there is [no __get_cpu_var defined](http://lxr.free-electrons.com/ident?v=3.19;i=__get_cpu_var).

This is [make.log](https://gist.github.com/Alukardd/dd3dc6445e04984be815) from my system.

It will be great if you fix netflow module for linux-3.19 and later...

**Comment by aabc** [comment]
3.19 compatibility should be already fixed in git head, can you try it?

**Comment by Alukardd** [comment]
No, I'' ll try only with latest release — v2.1. Is head is stable for production use?

**Comment by aabc** [comment]
If you don't enable latest promisc, vlan, and mpls features, it should almost same as 2.1. And, there is no bugreports for other improvements for several months (since Feb). Thus, it should be quite stable.

(I still would recommend to configure hardware watchdog even for a stable code for mission critical server, just in case.)

**Comment by Alukardd** [comment]
Ok, thanks. I'll try to use head later...
